benefits/core/templates/core/landing.html

```diff
@@ -11,6 +11,7 @@
         <div class="box px-4 py-3">
           <h1 class="text-left p-0">{{ page.headline }}</h1>
           <h2 class="p-sm pb-lg-8 pt-1 pt-lg-4 pb-4">{% translate "core.pages.landing.h2" %}</h2>
+          <span role="img" aria-label="{% translate "core.pages.index.alt" %}"></span>
```

Comment:
This also has the added benefit of inlining the alt text in the most appropriate place for the user, in context with the other (primary) content on the page.